.github/workflows/build-and-test.yml

```diff
@@ -48,20 +48,18 @@ jobs:
         CC: ${{ matrix.compiler }}
         LIBDQLITE_TRACE: 1
       run: |
-          for bin in unit-test integration-test \
-                     raft-core-fuzzy-test raft-core-integration-test \
-                     raft-core-unit-test raft-uv-integration-test \
-                     raft-uv-unit-test
-          do ./$bin || touch any-failed
-          done 2>&1 | tee -a test-suite.log
-          test '!' -e any-failed
+          tests="integration-test \
+                 raft-core-fuzzy-test \
+                 raft-core-integration-test \
+                 raft-core-unit-test \
+                 raft-uv-integration-test \
+                 raft-uv-unit-test"
+          make check TESTS="$tests"
 
-    - name: OnTestFailure
-      if: ${{ failure() }}
-      uses: actions/upload-artifact@v3
-      with:
-        name: test-suite.log
-        path: test-suite.log
+          ./unit-test --no-fork &
+          pid=$!
+          bash -c "sleep 10m; sudo gdb -p $pid -batch -ex 'thread apply all bt' -ex quit; false" &
```

Comment:
Are you sure gdb is in the container image?

Reply:
It is not, good catch! Fixed.